plugins/sfc/renderer/api.go

```diff
@@ -148,6 +149,11 @@ type PodSF struct {
 	// without further processing. Non-local pods contain logical names as they came from CRD.
 	InputInterface  string // name of the interface trough which the traffic enters the pod
 	OutputInterface string // name of the interface using which the traffic leaves the pod
+
```

Comment:
cc @milanlenco 

I'm not sure about this API / naming: `InputInterface` vs  `InputInterfaceConfigName` (and the same for output) for the virtual interface name defined in CRD vs logical name used for config. We may think about something else here.

Maybe `InputInterface` & `OutputInterface ` could be a struct, with members `CrdName` & `LogicalName`, or something similar?

Anyway, I would not name the CRD as "configuration file" in the comment, that may be misleading. It may not necessarily be any file.

Reply:
fixed in https://github.com/contiv/vpp/pull/1663/commits/291b06f116853535bb5c1f0a35b90e4fd9ed863b (sorry for big commit)